src/nrnmpi/Makefile.am

```diff
@@ -1,12 +1,12 @@
 lib_LTLIBRARIES = libnrnmpi.la
 
 nsrc=$(top_srcdir)/src
-includes = -I$(nsrc)/oc @NRN_REALTIME_INCLUDE@ -I@MUSIC_INCDIR@
-AM_CFLAGS = @MINGW_CFLAG@ $(includes)
+includes = -I$(nsrc)/oc @NRN_REALTIME_INCLUDE@ -I@MUSIC_INCDIR@ -I$(nsrc)/ivos
+AM_CPPFLAGS = $(includes)
 
-libnrnmpi_la_SOURCES = nrnmpi.c bbsmpipack.c mpispike.c nrnrt.c
+libnrnmpi_la_SOURCES = nrnmpi.cpp bbsmpipack.cpp mpispike.cpp nrnrt.cpp
 
-CC = @MPICCnrnmpi@
+CXX = @MPICCnrnmpi@
```

Comment:
Will check. Although we'd want to avoid linking to c++ bindings. In CMake I put : 
```
  # avoid linking to C++ bindings
  add_definitions("-DMPI_NO_CPPBIND=1")
  add_definitions("-DOMPI_SKIP_MPICXX=1")
  add_definitions("-DMPICH_SKIP_MPICXX=1")
```

Reply:
That worked, thanks for directing me.